### DISCUSSION TITLE
VALLHORN Wireless Motion Sensor flagged as unsupported device

### DISCUSSION THREAD

**Discussion by danielo515** [Koenkk/zigbee2mqtt#23636] [General]
I'm getting this in my logs:

```
warning Received message from unsupported device with Zigbee model 'VALLHORN Wireless Motion Sensor' and manufacturer name 'IKEA of Sweden'
debug Received Zigbee message from 'motion_8', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":1,"fileVersion":16777316,"imageType":6456,"manufacturerCode":4476}' from endpoint 1 with groupID 0
```

I also see this in the log if I see the file directly:
```
error 2024-08-16 10:54:13: Failed to interview '0x8c65a3fffe0dcc9d', device has not successfully been paired
info  2024-08-16 10:54:13: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x8c65a3fffe0dcc9d","ieee_address":"0x8c65a3fffe0dcc9d","status":"failed"},"type":"device_interview"}'
info  2024-08-16 10:54:13: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_failed","meta":{"friendly_name":"0x8c65a3fffe0dcc9d"},"type":"pairing"}'
debug 2024-08-16 10:55:55: Saving state to file /config/zigbee2mqtt/state.json
```

As far as I know, this is one of the [officially supported devices](https://www.zigbee2mqtt.io/devices/E2134.html). What I am doing wrong? What else I can do?
I'm using 2 AA rechargeable batteries as instructed in the device page.

**Comment by Koenkk** [comment]
Try updating to z2m 1.39.1

**Comment by danielo515** [comment]
I'm using the home assistant integration. How do I do that?

**Reply by Koenkk** [reply]
You have to update to the latest version of the addon (which is 1.39.1 atm)

**Reply by danielo515** [reply]
That is weird because I have no update available and this is what I have:
```
Zigbee2mqtt
Current version: 1.18.1-1 (Changelog)
```

**Reply by Koenkk** [reply]
The addon has been moved to a new repo, see https://github.com/zigbee2mqtt/hassio-zigbee2mqtt

**Reply by danielo515** [reply]
I finally linked the device with my new ikea hub (which I can position better than my central zigbee hub).
However, the app seems to work better now with many of the other devices, so I consider this fixed

**Comment by danielo515** [comment]
Thank you. I updated and things are looking good.
Let's see if it works.
Thanks

On Mon, Aug 19, 2024 at 9:00 PM Koen Kanters ***@***.***>
wrote:

> The addon has been moved to a new repo, see
> https://github.com/zigbee2mqtt/hassio-zigbee2mqtt
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Koenkk/zigbee2mqtt/discussions/23636#discussioncomment-10387654>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARKJWMKXJHV3USQVXK5JPLZSI6E5AVCNFSM6AAAAABMT55GO2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZYG43DKNA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
---
https://danielorodriguez.com
